src/schema/v2/artwork/index.ts

```diff
@@ -1515,10 +1519,22 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
             _.sortBy(typedImages, "position")
           )
 
-          const typedVideos = VIDEOS[id]
-            ? [{ ...VIDEOS[id], type: "Video" }]
+          const typedVideos = external_video_id
+            ? [
+                {
+                  type: "Video",
+                  url: external_video_id,
+                  width: 360,
+                  height: 360,
+                },
+              ]
             : []
-          return [...sortedTypedImages, ...typedVideos]
+
+          if (set_video_as_cover) {
+            return [...typedVideos, ...sortedTypedImages]
+          } else {
+            return [...sortedTypedImages, ...typedVideos]
+          }
```

Comment:
**nitpick [non-blocking]:** add missing test

I think this little bit here is worth a test just because it's a bit odd so maybe future us will have a better time remembering what's going on here with a spec for before/after and whether the video array is empty or not. What do you think?

*format via [Conventional Comments](https://conventionalcomments.org/)*

Reply:
Will add!